backend/dataall/modules/datapipelines/cdk/__init__.py

```diff
@@ -1,11 +1,13 @@
 from dataall.modules.datapipelines.cdk import datapipelines_cdk_cli_wrapper_extension, datapipelines_cdk_pipeline, \
-    datapipelines_stepfunctions_policy, datapipelines_lambda_policy, datapipelines_codestar_policy, datapipelines_pipeline
+    env_role_datapipelines_stepfunctions_policy, env_role_datapipelines_lambda_policy, env_role_datapipelines_cicd_policy, datapipelines_pipeline, \
+    pivot_role_datapipelines_policy
 
 __all__ = [
     'datapipelines_cdk_cli_wrapper_extension',
     'datapipelines_cdk_pipeline',
-    'datapipelines_stepfunctions_policy',
-    'datapipelines_lambda_policy',
-    'datapipelines_codestar_policy',
-    'datapipelines_pipeline'
+    'env_role_datapipelines_stepfunctions_policy',
+    'env_role_datapipelines_lambda_policy',
+    'env_role_datapipelines_policy',
```

Comment:
There is no `env_role_datapipelines_policy` in the directory. The file should be adjusted